### ISSUE TITLE
[Info] Accessibility info on Toasts 

### ISSUE THREAD

**Issue by bommariusser** [kirbydesign/designsystem#574]
**Is your enhancement request related to a problem? Please describe.**
We want all components to be accessible and the question was: Are Toasts Accessible, especially for screen readers, and how to get them accessible.

There is quite a decent amount of information of toasts in regards to WCAG and in short; Yes they should and can be accessible:

See also article:
https://medium.com/@sheribyrnehaber/designing-toast-messages-for-accessibility-fb610ac364be

**

> New to WCAG 2.1, satisfying Guideline 4.1.3 requires that:
> In content implemented using markup languages, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus.
> Using ARIA techniques such as role alert and aria-live, toast messages can be made available for screen reading technologies as soon as they are displayed.

**

and: https://www.scottohara.me/blog/2019/07/08/a-toast-to-a11y-toasts.html

**

> A toast component should be considered a type of status message, and thus should be given a role="status". 
> This will ensure that when a toast is displayed on screen, its contents will be, politely, communicated to assistive technologies, such as screen readers.

**
